docs/content/stable/drivers-orms/smart-drivers-ycql.md

```diff
@@ -0,0 +1,107 @@
+---
+title: YugabyteDB smart drivers for YCQL
+linkTitle: Smart drivers
+description: Use YugabyteDB smart drivers to improve performance with partition-aware load balancing and JSON support for YCQL
+headcontent: Manage partition-aware load balancing automatically using smart drivers
+menu:
+  stable:
+    identifier: smart-drivers-ycql
+    parent: drivers-orms
+    weight: 400
+type: docs
+---
+
+{{<tabs>}}
+{{<tabitem href="../smart-drivers/" text="YSQL" icon="postgres" >}}
+{{<tabitem href="../smart-drivers-ycql/" text="YCQL" icon="cassandra" active="true" >}}
+{{</tabs>}}
+
+The standard/official drivers available for Cassandra work out-of-the-box with [YugabyteDB YCQL API](../../api/ycql/).
+
+In addition to the compatible Cassandra drivers, YugabyteDB also developed the following YCQL drivers with specific changes to leverage YugabyteDB features, available as open source software under the Apache 2.0 license.
+
+| GitHub project | Based on | Learn more |
+| :--- | :--- | :--- |
+| [YugabyteDB Java Driver for YCQL (3.10)](https://github.com/yugabyte/cassandra-java-driver) | [DataStax Java Driver 3.10](https://docs.datastax.com/en/developer/java-driver/3.10/) | [Documentation](../java/ycql/) |
+| [YugabyteDB Java Driver for YCQL (4.15)](https://github.com/yugabyte/cassandra-java-driver/tree/4.15.x) | [DataStax Java Driver 4.15](https://docs.datastax.com/en/developer/java-driver/4.15/) | [Documentation](../java/ycql-4.x/) |
+| [YugabyteDB Go Driver for YCQL](https://github.com/yugabyte/gocql) | [gocql](https://gocql.github.io/) | [Documentation](../go/ycql/) |
+| [YugabyteDB Python Driver for YCQL](https://github.com/yugabyte/cassandra-python-driver) | [DataStax Python Driver](https://github.com/datastax/python-driver) | [Documentation](../python/ycql/) |
+| [YugabyteDB Node.js Driver for YCQL](https://github.com/yugabyte/cassandra-nodejs-driver) | [DataStax Node.js Driver](https://github.com/datastax/nodejs-driver) | [Documentation](../nodejs/ycql/) |
+| [YugabyteDB C++ Driver for YCQL](https://github.com/yugabyte/cassandra-cpp-driver) | [DataStax C++ Driver](https://github.com/datastax/cpp-driver) | [Documentation](../cpp/ycql/) |
+| [YugabyteDB C# Driver for YCQL](https://github.com/yugabyte/cassandra-csharp-driver) | [DataStax C# Driver](https://github.com/datastax/csharp-driver) | [Documentation](../csharp/ycql/) |
+| [YugabyteDB Ruby Driver for YCQL](https://github.com/yugabyte/cassandra-ruby-driver) | [DataStax Ruby Driver](https://github.com/datastax/ruby-driver) | [Documentation](../rust/yb-rust-postgres) |
+
+All YugabyteDB smart driver libraries are actively maintained, and receive bug fixes, performance enhancements, and security patches.
+
+## Key features
+
+YugabyteDB smart drivers have the following key features.
+
+| Feature | Notes |
+| :--- | :--- |
+| [Retry policy](#retry-policy) | Smart driver retries certain operations in case of failures. |
```

Comment:
```suggestion
| [Retry policy](#retry-policy) | Like the upstream driver, the YugabyteDB YCQL driver retries certain operations in case of failures. |
```